### PR TITLE
chore: Add build stats for NPM package

### PR DESCRIPTION
**Pull request by patrickhousley**
### Overview

Adding PR commenting for NPM package consumption.

### Related Issue(s)

https://issues.newrelic.com/browse/NEWRELIC-8077

### Testing

Should comment on this PR.

### CODE REVIEW

**Review by metal-messiah — APPROVED** [review]

File: tools/scripts/diff-sizes.mjs [hunk 1]
```diff
@@ -82,6 +89,7 @@ async function getBuildSize (buildType) {
   }
 
   const buildStats = await fs.readJson(buildStatsPath)
+  if (buildType === 'npm') console.log(buildStatsPath)
```
Comment (bjfield) [inline tools/scripts/diff-sizes.mjs:92]:
```suggestion
```

Comment (bjfield) [reply tools/scripts/diff-sizes.mjs:92]: I'm thinking this was a leftover from testing stuff. If not, curious why it's logging only for 'npm' build type.

File: tools/scripts/diff-sizes.mjs [hunk 2]
```diff
@@ -139,16 +147,22 @@ async function writeDiff (assetSizes) {
...
-Merging this pull request will result in the following asset size changes:
+Merging this pull request will result in the following CDN asset size changes:
 
 | Asset Name | Previous Size | New Size | Diff |
 |------------|---------------|----------|------|
 ${generateDiffRows(assetSizes, 'standard')}
 ${generateDiffRows(assetSizes, 'polyfills')}
 ${generateDiffRows(assetSizes, 'worker')}
 
+Merging this pull request will result in the following NPM package consumer size changes:
```
Comment (bjfield) [inline tools/scripts/diff-sizes.mjs:158]:
```suggestion
Merging this pull request will result in the following **NPM package consumer size changes**:
```

File: tools/scripts/diff-sizes.mjs [hunk 3]
```diff
@@ -139,16 +147,22 @@ async function writeDiff (assetSizes) {
   await fs.ensureDir(config.out)
   await fs.writeJson(path.join(config.out, 'size_report.json'), assetSizes, { spaces: 2 })
   await fs.outputFile(path.join(config.out, 'size_report.md'), `# Asset Size Report
-Merging this pull request will result in the following asset size changes:
+Merging this pull request will result in the following CDN asset size changes:
```
Comment (bjfield) [inline tools/scripts/diff-sizes.mjs:150]:
```suggestion
Merging this pull request will result in the following **CDN asset size changes**:
```

File: tools/scripts/diff-sizes.mjs (hunk 1 above)
Comment (patrickhousley) [reply tools/scripts/diff-sizes.mjs:92]: TY

**Pull request merged**